server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -496,7 +496,7 @@ private List<PodSpec> installPods(File workingDir) throws IOException, ExtenderE
             throw new ExtenderException("Unable to find Podfile " + podFile);
         }
         File dir = podFile.getParentFile();
-        String log = execCommand("pod install --verbose", workingDir);
+        String log = execCommand("pod install  --repo-update --verbose", workingDir);
```

Comment:
How does this work with different versions of pods?
E.g. project A wants pod Foo at version 1.1, whereas project B wants version 2.1 ?

Reply:
It's like with any tool for resolving dependencies. Cocoapods apply a set of rules when dependencies are resolved where you can specify exact versions or minimum versions or a range of acceptable versions or no version, and Cocoapods will do its best to satisfy this. If it fails it will write this to the log and the build will not proceed.

https://guides.cocoapods.org/using/the-podfile.html#specifying-pod-versions